docs/cache/integrations/redis-client-compatibility.md

```diff
@@ -156,6 +156,32 @@ redis_client = MomentoRedis(
 
 For more in-depth information, with example code, please see [Momento Python Redis compatibility client](https://github.com/momentohq/momento-python-redis-client) on GitHub.
 
+</TabItem>
+<TabItem value="lettuce" label="lettuce" default>
+
+```java
+import io.lettuce.core.api.reactive.RedisReactiveCommands;
+import java.time.Duration;
+import momento.lettuce.MomentoRedisReactiveClient;
+import momento.sdk.CacheClient;
+import momento.sdk.auth.CredentialProvider;
+import momento.sdk.config.Configurations;
+
+class Basic {
```

Comment:
Any thoughts on the name here?

Reply:
`CompatibilityExample`?

Reply:
renamed in [0f9487f](https://github.com/momentohq/public-dev-docs/pull/1317/commits/0f9487fdf1c8505c929463623284226acc5ffafb)